recipe_batches/recipe_batches.py

```diff
@@ -2,13 +2,42 @@
 
 import math
 
+ingredients = {
+  'eggs': 15,
+  'butter': 30,
+  'sugar': 24,
+  'flour': 45
+}
+
+recipe = {
+  'eggs': 7,
+  'butter': 10,
+  'sugar': 10,
+  'flour': 5
+}
+
 def recipe_batches(recipe, ingredients):
-  pass 
+  recipeValues = []
+  ingredientValues = []
+  dividedValues = []
+  for x in recipe.values():
```

Comment:
Nice job solving this problem. A way to solve this problem without 3 for loops is like this - 

min_ratio = math.inf (infinity)

for ingredient, amount in recipe.items():
____if ingredient not in ingredients:
________return 0
____ratio = math.floor(ingredients[ingredient] / amount)
____if ratio < min_ratio:
________min_ratio = ratio
    
return min_ratio